usagov/vote-gov-drupal
# VOTE-2791 add view revision permissions to content manager and conten…

…t editor



## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-2791

## Description

Update content role permissions for revisions

## Deployment and testing

### Post-deploy steps

1. `lando retune`

### QA/Testing instructions

1. Log in as a Content Manager and make sure you can view revisions for terms and revert.
3. Log in as a Content Editor and make sure you can view revisions for terms but not revert.

## Checklist for the Developer




## Checklist for the Peer Reviewers
